src/datastores/handlers/base.js

```diff
@@ -148,11 +148,34 @@ class Playlists {
   }
 }
 
+class ChannelBlocker {
```

Comment:
I personally don't think that a feature like this needs to be separated into a brand new database / file. Especially when it stores very little in compared to the other files. This would be fine to be stored in the settings database as an array of channels. Please migrate the DB to the settings DB

Reply:
I have 600 blocked channels on the browser extension.
Not everyone will have a large list but is it ok to be in `settings.db` if it's 50~100 or so?